.markdownlint.yaml

```
@@ -34,3 +42,7 @@ MD040: true
 # MD042/no-empty-links - No empty links
 MD042: true
 
+# MD048/code-fence-style - Code fence style
+MD048:
+  # Code fence syle
+  style: "consistent"
```

```suggestion
  style: "backtick"
```

As the rule states, a consistent style aids readability. I have never seen tilde-style fences in this repo, and I don't see any particular reason we would want to introduce them.